enrich/gpx/src/enrich.gpx.plugin.ts

```
@@ -0,0 +1,258 @@
+      })),
+    })) || []
+
+  let tabularData = convertToTabularFormat(waypoints, tracks, routes)
+
+  if (removeDuplicates) {
+    tabularData = removeDuplicatePoints(tabularData)
+  }
+
+  if (
+    filterDates &&
+    !isNaN(startDate.getTime()) &&
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)